skills/scribe/scripts/audit.py

```
@@ -0,0 +1,242 @@
+    return findings
+
+
+def check_between_files(docs):
+    """Content lost where one file was cut and another picks up.
+
+    This is the failure that produces a summary reading as complete when a
+    whole stretch of the meeting was never recorded.
+    """
+    findings = []
+    ordered = sorted(docs, key=lambda d: d.get("audio", {}).get("path", ""))
+    for cur, nxt in zip(ordered, ordered[1:]):
```

Ruff (0.16.0)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)